integration-test/__init__.py

```diff
@@ -1194,7 +1230,19 @@ def load_fixtures(self, urls, clip, simplify):
                     "very large, %d bytes." \
                     % (self.id(), geojson_size)
 
-        feature_fetcher = FixtureFeatureFetcher([geojson_file], self.env)
+        rows, rels = _load_fixtures([geojson_file])
+        feature_fetcher = FixtureFeatureFetcher(rows, rels, self.env)
+        self.assertions = Assertions(feature_fetcher, self.test)
+
+    def generate_fixtures(self, objs):
+        # features in rows will be dsl.Feature objects (which are namedtuples)
+        # or tuples of (fid, shape, properties). the rels should be dicts with
+        # keys for id, tags, way and rel offsets and "parts" array of IDs, as
+        # if they had come from osm2pgsql's planet_osm_rels table.
+        rows = filter(lambda o: isinstance(o, tuple), objs)
+        rels = filter(lambda o: isinstance(o, dict), objs)
```

Comment:
I'm surprised the hateful flake8 let this slide :)

Reply:
Good point! Do you think it would be better as

```python
rows = []
rels = []
for o in objs:
    if isinstance(o, tuple):
        rows.append(o)
    elif isinstance(o, dict):
        rels.append(o)
```
?

Reply:
Yea, that looks good to me.

Reply:
Cool, thanks. Fixed in 2129454.